<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>240</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="asg0_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>ASG0 / OUT1</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="waveform_layout">
       <item>
        <widget class="QLabel" name="label_asg_fct_3">
         <property name="text">
          <string>Waveform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="waveform0_box">
         <item>
          <property name="text">
           <string>dc</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ramp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>halframp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>square</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>noise</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_ampl_4">
         <property name="text">
          <string>Amplitude [Vp]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="amplitude0_spinbox">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000122000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_ampl_2">
         <property name="text">
          <string>Offset [V]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="offset0_spinbox">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000122000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_ampl_3">
         <property name="text">
          <string>Frequency [Hz]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="frequency0_spinbox">
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>62500000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.116415000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
